src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -683,6 +683,22 @@ internal virtual bool SupportsExtensionMethods
             Binder originalBinder,
             bool diagnose,
             ref HashSet<DiagnosticInfo> useSiteDiagnostics)
+        {
+            LookupMembersInClass(result, type, name, arity, basesBeingResolved, options, originalBinder, type, diagnose, ref useSiteDiagnostics);
+        }
+
+        // Lookup member in a class, struct, enum, delegate.
```

Comment:
>delegate [](start = 51, length = 8)

, interface

Reply:
> , interface

I do not think so.

---
In reply to: [271072816](https://github.com/dotnet/roslyn/pull/34579#discussion_r271072816) [](ancestors = 271072816)